### PR TITLE
data: nitishkumar07

### PR DESCRIPTION
**Pull request by nitishkumar07**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots
screenshot1

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2937"></a>

### CODE REVIEW

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

File: data/nitishkumar07.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  "name": "Nitishkumar J G",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Student @ RNSIT",
+  "avatar": "https://github.com/nitishkumar07.png",
+  "links": [
+    {
+      "name": "Connect with me on Linkedin",
+      "url": "https://www.linkedin.com/in/nitishkumarjg/",
+      "icon": "faLinkedin"
```
Comment (eddiejaoude) [inline data/nitishkumar07.json:11]: The icon needs to be `FaLinkedin`, you can check available icons here https://linkfree.eddiehub.io/icons in the docs

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Icon used is not valid

**Comment by nitishkumar07** [comment]
I have done the requested changes @eddiejaoude

**Review by Panquesito7 — APPROVED** [review]

**Review by eddiejaoude — APPROVED** [review]

**Pull request merged**